qubesmanager/qube_manager.py

```diff
@@ -715,9 +719,25 @@ class VmManagerWindow(ui_qubemanager.Ui_VmManagerWindow, QMainWindow):
     settings_loaded = False
 
     def __init__(self, qt_app, qubes_app, dispatcher, _parent=None):
+    # pylint: disable-msg=too-many-statements
         super().__init__()
         self.setupUi(self)
 
+        # migrate old config file to the unified config directory
+        # TODO: Remove this block when Qubes R4.3 becomes End-of-Life
+        old_config_home = xdg.BaseDirectory.xdg_config_home + \
+            '/The Qubes Project'
+        new_config_home = xdg.BaseDirectory.xdg_config_home + '/qubes-os'
+        config_file = 'qubes-qube-manager.conf'
+        old_config_file = old_config_home + '/' + config_file
+        new_config_file = new_config_home + '/' + config_file
+        if path.isfile(old_config_file) and not path.isfile(new_config_file):
+            pathlib.Path(new_config_home).mkdir(parents=True, exist_ok=True)
+            shutil.copy(old_config_file, new_config_file)
+
+        QCoreApplication.setOrganizationName('qubes-os')
```

Comment:
This actually duplicates code from utils.py (`run_asynchronous` and `run_synchronous` functions, where the general setup for QApplications is made. I think that file would be a better place for changes like this? And generally, maybe if we put sensible org name there, and the migration code also there, it could be only done in one place and not in multiple places (for other manager tools).

Reply:
It is done. The code is transferred to `utils.py` as advised and wrapped inside a function (so it would look nicer). The call to the said function is also done within the module.

Reply:
> I would propose to have this function be called in run synchronous/ run asynchronous helper functions so that when other tools from this package start using QSettings, it does not end in ressurection of The Qubes Project directory

It is done. Calls are from `run_synchronus` and `run_asynchronous` function.

> and conversely, to remove the old setting of app name from lines 576 and 536.

I wonder why I missed them. They are now deleted.